index.html

```diff
@@ -65,6 +68,22 @@ <h1>tsParticles 404 Templates</h1>
     </div>
   </main>
 
+  <!-- PRELOADER JS-->
+  <!-- preloader script tag -->
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.6.1/jquery.min.js"></script>
+  <!--preloader JS -->
+  <script>
```

Comment:
Move this JavaScript code, without comments, in the `js/main.js` file, or create a `js/preloader.js` file and include it in the page.